unAPI.js

```diff
@@ -109,89 +147,92 @@ UnAPIFormat = function(aXML) {
  * @constructor
  * @param {String} id The ID contained in an abbr tag
  */
-UnAPIID = function(id) {
+function UnAPIID(id) {
 	this.id = id;
 	unAPIIDs[id] = this;
-};
+}
 
 UnAPIID.prototype = {
+
 	/**
 	 * Gets the item type for this item
 	 * @param {Function} callback Callback to be passed itemType when it is known
 	 */
-	"getItemType":function(callback) {
-		var me = this;
-		this.getItems(function(items) {
+	getItemType: function (callback) {
+		this.getItems(function (items) {
 			if (items.length === 0) {
 				callback(false);
-			} else if (items.length === 1) {
+			}
+			else if (items.length === 1) {
 				callback(items[0].itemType);
-			} else {
+			}
+			else {
 				callback("multiple");
 			}
 		});
 	},
+
 	/**
 	 * Gets items associated with this ID
 	 * @param {Function} callback Callback to be passed items when they have been retrieved
 	 */
-	"getItems":function(callback) {
+	getItems: function (callback) {
 		if (this.items) {
 			callback(this.items);
 			return;
 		}
 		
-		var me = this;
```

Comment:
Why did you delete this variable and what was it doing in the first place?

Reply:
It brought a copy of `this` into the function passed to `isSupported` (otherwise `this` inside that function would be `null`), but arrow-functions retain the outer `this` so they don't need this trick.